app/components/AppUpdater/AppUpdateModal.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import React from 'react'
+import swal from 'sweetalert'
+import ReactDOM from 'react-dom'
+
+function getModalNode(link: string) {
+  const wrapper = document.createElement('div')
+  ReactDOM.render(<AppUpdateModal link={link} />, wrapper)
+  return wrapper.firstChild
+}
+
+const appUpdateModal = (link: string) => swal({
+  title: 'Update',
+  button: false,
+  content: getModalNode(link),
+})
+
+type Props = {
+  link: string
+};
+
+class AppUpdateModal extends React.Component<Props> {
+  onDismiss = () => {
+    swal.setActionValue({ cancel: true })
+    swal.close()
+  }
+
+  onDownload = () => swal.close()
```

Comment:
@thehogfather 

in order for a link in Electron to open in the regular browser you need to do something similar to how it is implemented in the Faucet.js file

const { shell } = require('electron')

  onLinkClick = (evt) => {
    evt.preventDefault()
    shell.openExternal(evt.target.href)
  }

otherwise it doesn't open the link

Reply:
@thehogfather maybe create an `ExternalLink` component to handle that logic?